pkg/volunteer/volunteer.go

```diff
@@ -103,12 +109,18 @@ func (v *volunteer) generateRecord() (report.Record, error) {
 		return report.Record{}, err
 	}
 
+	extensions, err := v.extensionsLister.ListExtensions()
+	if err != nil {
+		return report.Record{}, err
+	}
```

Comment:
That's a very good point. I'll modify this PR to not fail the report generation just because of failed extension generation.